docs/content/commands/rclone_mount.md

```diff
@@ -459,6 +459,12 @@ by `-` and prepend `--` to get the command-line flags. Options containing commas
 or spaces can be wrapped in single or double quotes. Any inner quotes inside outer
 quotes of the same type should be doubled.
 
+On FreeBSD you can symlink the binary with `ln -s /usr/local/bin/rclone /usr/local/bin/rclonefs` 
```

Comment:
This change needs to be in the source file which is [cmd/mountlib/mount.md](https://github.com/rclone/rclone/blob/master/cmd/mountlib/mount.md) as this file is autogenerated from it.